src/main/resources/swagger/api-docs.yaml

```diff
@@ -2084,6 +2084,73 @@ paths:
             - email
             - profile
             - https://www.googleapis.com/auth/devstorage.full_control
+  '/api/library/{workspaceNamespace}/{workspaceName}/published':
+    put:
+      tags:
+        - Library
+      operationId: publishLibraryWorkspace
+      summary: |
+        publish the workspace in the Library
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+      responses:
```

Comment:
Looks like this can return a "forbidden" as well. Also, since pretty much any endpoint can return a 500, is that worth including in our API docs for every single URL?